### PR TITLE
Change passport auth block

### LINKED ISSUE #91: [1] Move passport to using auth[*] fields for login tests

### PR DESCRIPTION
**Pull request by tschuy**
Fixes #91

### CODE REVIEW

**Comment by Kennric** [comment]
Looks correct, I like the auth {} block structure, I think that will help keep things flexible and extendable. +1

**Comment by iankronquist** [comment]
@tschuy Do you think you should use `knex.first`?

**Comment by tschuy** [comment]
Yeah, I'll get that.

File: src/auth/local.js [hunk 1]
```diff
@@ -3,6 +3,10 @@ var bcrypt = require('bcrypt');
 
 module.exports = function(knex) {
     return new LocalStrategy(
+        {
+            usernameField: 'auth[username]',
+            passwordField: 'auth[password]'
+        },
```
Comment (ElijahCaine) [inline src/auth/local.js]: @tschuy What do these two lines do with usernameField and passwordField?

Comment (tschuy) [inline src/auth/local.js]: https://github.com/jaredhanson/passport-local#parameters

**Comment by ElijahCaine** [comment]
Looks good to me. +1

**Pull request merged**